study-builder/studies/testboston/i18n/es.conf

```diff
@@ -160,6 +160,10 @@
     "name": "Es hora de recibir otro kit",
     "title": "Encuesta de seguimiento",
   },
+  "symptoms_survey": {
+    "name": "Encuesta de síntomas de referencia",
```

Comment:
I'll recommend NOT putting in Spanish/Creole strings until we get the official translations. What I did was put in the English as a placeholder and tagging it with `[ES]` or `[HT]` so we have a sense of what's left to translate.

Reply:
hmm. 
I spent extra time using google translator to check each prompt/text in "es" and "ht" and filling it in. :confused: